grakn-engine/src/main/java/ai/grakn/engine/tasks/config/ConfigHelper.java

```diff
@@ -18,7 +18,7 @@
 
 package ai.grakn.engine.tasks.config;
 
-import ai.grakn.engine.tasks.TaskId;
+import ai.grakn.engine.TaskId;
```

Comment:
Yeah. There is the whole question of "should internal-engine classes be in `core`?" to also consider. Although maybe there are no longer internal engine classes.